### PR TITLE
Inspector test

### PR DESCRIPTION
**Pull request by darkwing**
**Description:**
Adds test for Inspector

**Changes proposed:**
None

### CODE REVIEW

File: tests/components/scene/inspector.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  teardown(function () {
+    var el = this.sceneEl;
+    el.parentNode.removeChild(el);
+  });
+
+  test('adds inspector SCRIPT element to page', function (done) {
+    var el = this.sceneEl;
+
+    el.addEventListener('loaded', function () {
```
Comment (ngokevin) [inline tests/components/scene/inspector.test.js]: We could call this in the setup

**Comment by ngokevin** [comment]
We can get full coverage if we also test the `remove` (detached) callback. This should remove the event listener when the components is detached (with `removeAttribute`).

File: tests/components/scene/inspector.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      var i;
+      var matchingScripts = 0;
+      var scripts;
+
+      // Have to call method directly because Chrome doesn't provide
+      // a reliable method for KeyboardEvent
```
Comment (ngokevin) [inline tests/components/scene/inspector.test.js]: Maybe we should emit a CustomEvent on `window`?

File: tests/components/scene/inspector.test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      el.components.inspector.onKeydown({
+        keyCode: 73,
+        ctrlKey: true,
+        altKey: true
+      });
+
+      scripts = document.querySelectorAll('script');
```
Comment (ngokevin) [inline tests/components/scene/inspector.test.js]: something like `assert.ok(document.querySelector(script[src="..."]))` ?

Comment (darkwing) [inline tests/components/scene/inspector.test.js]: Illegal selector :/

Comment (ngokevin) [inline tests/components/scene/inspector.test.js]: We could add `data-name="aframe-inspector" aframe-injected` attributes to the injected script tag.

**Comment by darkwing** [comment]
Updates:

1.  Added the attributes you suggested; I held off originally to avoid code bloat for sake of test but since we're already using `aframe-injected` elsewhere, probably good to be consistent.

2.  Not adding `remove()` test since default components cannot be removed at present.

**Comment by ngokevin** [comment]
Thanks!!

**Pull request merged**